spring-boot/admin/src/main/resources/application.yml

```diff
@@ -76,6 +86,14 @@ spring:
   data:
     rest:
       base-path: /api
+  thymeleaf:
+    # Enables reloading of thymeleaf templates on runtime using IntelliJ
```

Comment:
We should not put IDE-specific settings in the general config file. Could maybe be put into the "private profile template file".

Reply:
Actually, only the comment is IDE specific, the configuration is not so I just removed the "using IntelliJ" part. 

I would like to keep that in the default yml to have a nice out-of-the-box working setup for developers which enables resource reloading, especially for frontend works. I don't see a disadvantage in this approach, yet. Do you see any?

Reply:
If this is just for developers, maybe move it to the private application yml template? You would not want this setting for a normal installation.

But this is not so important for now, could also stay as it is and we could discuss reasonable defaults for different situations as a follow-up.

Reply:
Done. I just moved it to my private profile as suggested.